keyboards/mechmini/v2/keymaps/arkag/config.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#define TAPPING_TERM                200
```

Comment:
This stuff is in both your userspace and here.  You may want to remove one of the sets of defines.